tests/commands/logs.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+const { spawnSync } = require('child_process');
+
+import {
+  CLASP,
+  SCRIPT_ID,
+} from '../constants';
+
+import {
+  cleanup,
+  setupWithoutGCPProject,
+} from '../functions';
+
+describe('Test clasp logs function', () => {
+  before(setupWithoutGCPProject);
+  it('should prompt for logs setup', () => {
+    const result = spawnSync(
+      CLASP, ['logs'], { encoding: 'utf8' },  // --setup is default behaviour
```

Comment:
If `--setup` is the default, why does this return different output from the 2nd test case below?

Reply:
It looks like it may have been default behavior when these tests were written, but the logic is different now:

https://github.com/google/clasp/blob/190480c9bfbdad4b944fbb0a9c1c17f273d128b8/src/commands/logs.ts#L143-L150

It looks like `--setup` is not default, but if there is no `projectId` in the `.clasp.json` then it exhibits the same behavior as `--setup`.

I think these tests should expect the same result, while testing different situations that lead to it:

https://github.com/google/clasp/blob/190480c9bfbdad4b944fbb0a9c1c17f273d128b8/src/commands/logs.ts#L106-L115